x/datadeal/keeper/reward_test.go

```diff
@@ -85,6 +85,55 @@ func (suite *rewardTestSuite) BeforeTest(_, _ string) {
 	})
 }
 
+func (suite rewardTestSuite) TestSellerRewardDistribution() {
+	ctx := suite.Ctx
+
+	err := suite.FundAccount(ctx, suite.buyerAccAddr, suite.defaultFunds)
+	suite.Require().NoError(err)
+
+	dealAddr := types.NewDealAddress(1)
+	msgCreateDeal := &types.MsgCreateDeal{
+		DataSchema:   []string{"http://jsonld.com"},
+		Budget:       &sdk.Coin{Denom: assets.MicroMedDenom, Amount: sdk.NewInt(100_000_000)}, // 100 MED,
+		MaxNumData:   1,
+		BuyerAddress: suite.buyerAccAddr.String(),
+	}
+
+	buyer, err := sdk.AccAddressFromBech32(suite.buyerAccAddr.String())
+	suite.Require().NoError(err)
+
+	dealID, err := suite.DataDealKeeper.CreateDeal(ctx, buyer, msgCreateDeal)
+	suite.Require().NoError(err)
+	suite.Require().Equal(uint64(1), dealID)
+
+	// balance check before reward distribution
+	dealBalance := suite.BankKeeper.GetBalance(ctx, dealAddr, assets.MicroMedDenom)
+	suite.Require().Equal(sdk.NewCoin(assets.MicroMedDenom, sdk.NewInt(100_000_000)), dealBalance)
+
+	sellerBalance := suite.BankKeeper.GetBalance(ctx, suite.sellerAccAddr, assets.MicroMedDenom)
+	suite.Require().Equal(sdk.NewCoin(assets.MicroMedDenom, sdk.NewInt(0)), sellerBalance)
+
+	dataSale := types.NewDataSale(&types.MsgSellData{
+		DealId:        1,
+		VerifiableCid: "verifiableCID",
+		SellerAddress: suite.sellerAccAddr.String(),
+	})
+
+	// distribute data verification rewards
+	err = suite.DataDealKeeper.DistributeVerificationRewards(ctx, dataSale)
+	suite.Require().NoError(err)
+
+	// TODO check balance of oracles
```

Comment:
How about adding a test to verify the current Oracle's balance?
If the Oracle reward distribution logic is added later, this test will fail.

Reply:
Right. If the distribution of oracle rewards is added, the balance of deal would be empty.
So I marked as TODO here.
I'll cover it when I add oracle reward distribution in the next PR.